tests/test_init.py

```diff
@@ -212,17 +212,12 @@ async def test_migration_from_version_11_to_12():
 
 async def test_setup_entry_coordinator_failure():
     """Test setup_entry when coordinator fails to update."""
+    from tests.const import FAKE_CONFIG_DATA
```

Comment:
Imports are only needed at the top of the file and only need to be imported once. :+1: